specs/disciplina/educator/api/educator.yaml

```diff
@@ -948,6 +1091,79 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Transaction'
+    Certificate:
+      type: object
+      required:
+        - id
+        - meta
+      properties:
+        id:
+          $ref: '#/components/schemas/Hash'
+        meta:
+          $ref: '#/components/schemas/CertificateMeta'
+    CertificateMeta:
+      type: object
+      required:
+        - student
+        - startYear
+        - endYear
+        - educationForm
+        - number
+        - issueDate
+        - title
+        - major
+      properties:
+        student:
+          type: object
+          required:
+            - name
+            - birthDate
+          properties:
+            name:
+              type: string
+            birthDate:
+              type: string
+              format: date
+        startYear:
+          type: integer
+          format: int32
+        endYear:
+          type: integer
+          format: int32
+        educationForm:
+          type: string
+          enum: [fulltime, parttime]
+        number:
+          type: string
+        issueDate:
+          type: string
+          format: date
+        title:
+          type: string
+        major:
+          type: string
+        specialization:
+          type: string
+    CertificateGrade:
+      type: object
+      required:
+        - subject
+        - grade
+      properties:
+        subject:
+          type: string
+        lang:
+          type: string
+          enum: [en, ru]
```

Comment:
Btw I wonder, do we promise to validate that passed values (enums, dates, e.t.c.) fit their format?

Reply:
JSON instances for those datatypes will ensure that.

Reply:
So we will parse those parameters as dedicated datatypes rather than just strings. Okay.